src/domains/marketing/Footer.js

```diff
@@ -31,7 +31,7 @@ export function Footer() {
           <a
             href="https://github.com/queen-raae/prune-your-follows/discussions/categories/support"
             className="underline"
-            onClick={() => (E5XIJ5CK, 0)}
+            onClick={() => trackGoal("E5XIJ5CK", 0)}
```

Comment:
suggestion: if we are to track clicks on these links to twitter and github they need to get their own link id

I created some:
GitHub Click	KPVE0RLO
Queen Raae Click	E5XIJ5CK
Twitter Click	0U7VHDJA
Xata Click	GEVKO638